cmd/manager/main.go

```diff
@@ -113,9 +113,10 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = serveCRMetrics(cfg); err != nil {
-		log.Info("Could not generate and serve custom resource metrics", "error", err.Error())
-	}
+	// After moving from namespace scoped to cluster scoped - Having this issue https://github.com/operator-framework/operator-sdk/issues/1858.
+	//if err = serveCRMetrics(cfg); err != nil {
```

Comment:
I wonder how it works with Tekton: https://github.com/tektoncd/operator/blob/master/cmd/manager/main.go#L128
They use cluster scoped CRD too.